ecs-agent/daemonimages/csidriver/driver/node.go

```diff
@@ -245,9 +252,40 @@ func (d *nodeService) NodeStageVolume(ctx context.Context, req *csi.NodeStageVol
 		}
 	}
 	klog.V(4).InfoS("NodeStageVolume: successfully staged volume", "source", source, "volumeID", volumeID, "target", target, "fstype", fsType)
+
+	SourceVolumeHostPath := target
+	if strings.HasPrefix(target, EBSPathPrefix) {
+		SourceVolumeHostPath = strings.TrimPrefix(target, EBSPathPrefix)
+	}
+
+	// Gid is generated based on SourceVolumeHostPath the same as in task.go
+	gid := util.GenerateGIDFromPath(SourceVolumeHostPath)
+	// Set permissions on the mount point to allow non-root users to access it
+	if err := setMountPointPermissions(target, gid); err != nil {
+		klog.Warningf("Failed to set permissions on mount point %s: %v", target, err)
+		// Continue even if permission setting fails, as the volume is still usable
```

Comment:
Its not usable by non-root user mode containers, right? In my opinion, we must not silently fail here.